content/blog/2019/2019-09-28-on-expected-vomiting-time.md

```diff
@@ -0,0 +1,90 @@
++++
+author="Janez Demšar"
+date= '2019-09-28'
+draft= true
+title="On Expected Vomiting Time"
+type="blog"
+thumbImage = "/blog_img/2019-09-28-vomiting-pumpkin.jpg"
+frontPageImage = "/blog_img/2019-09-28-vomiting-pumpkin.jpg"
+categories=["model performance", "confusion matrix"]
+shortExcerpt = "Computing meaningful performance scores of models should be creative"
+longExcerpt = "A report of an interesting ending of a lecture about setting probability thresholds for predictive models"
++++
+
+We just finished a lecture and a student came with a question. The lecture was, in short, about how predictive models compute probabilities, and to make decisions we must find a threshold that will minimize the cost, maximize the profit, strike a good balance between sensitivity and specificity, or increase the general human well-being in some other way. So this student came with a question about something I didn't explain well, and she gave me one last chance.
+
+I had some data that looked like this.
+
+| ID | predicted probabilities | аctual class |
+| -: | ----------------------: | ------------ | 
+|  1 | 0.984                   | p            |
+|  2 | 0.907                   | p            |
+|  3 | 0.881                   | n            |
+|  4 | 0.865                   | n            |
+|  5 | 0.815                   | p            |
+|  6 | 0.741                   | p            |
+|  7 | 0.735                   | p            |
+|  8 | 0.635                   | n            |
+|  9 | 0.582                   | n            |
+| 10 | 0.480                   | p            |
+| 11 | 0.413                   | n            |
+| 12 | 0.317                   | n            |
+| 13 | 0.287                   | n            |
+| 14 | 0.225                   | n            |
+| 15 | 0.216                   | p            |
+| 16 | 0.183                   | n            |
+
+
+The data is made up, so I had to make up its meaning to form an example. In the rush of a moment, I succumbed to classics: "*For each of those 16 people you predicted the probability for being sick. Whom are you going to give the cure?*" Her answer was in line with the best manners of modern medical practice (who cares about bacteria becoming resistant to antibiotics?): "*To all!*" I didn't expect this, but anyway planned to continue with emphasizing that this medicine is not harmless: "*People who take this medicine will vomit for one week.*" She turned by 180 degrees: "*Oh, no. To nobody!*" I didn't expect this either. I spend two weeks a year teaching in Moscow, and gosh I love these students! Her decision led nicely to my next condition: "*But if a person is indeed sick and you don't give him the drug, he'll vomit for four weeks.*"
+
+Decision making becomes much more interesting when both, yes and no, have consequences.
+
+So, to restate:
+- if you give the medicine to anybody, sick or healthy, he vomits for a week. 
```

Comment:
Add extra blank space before the list. That's a Hugo requirement or it doesn't get rendered correctly.